examples/tuning/cta_svm/main.py

```diff
@@ -42,27 +43,32 @@ def __call__(self, data):
     parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
     parser.add_argument("--cache", action="store_true", help="Cache processed data.")
     parser.add_argument("--dense_dim", type=int, default=400, help="dim of PCA")
-    parser.add_argument("--gpu", type=int, default=-1, help="GPU id, set to -1 for CPU")
+    parser.add_argument("--gpu", type=int, default=0, help="GPU id, set to -1 for CPU")
     parser.add_argument("--log_level", type=str, default="INFO", choices=get_args(LogLevel))
     parser.add_argument("--species", default="mouse")
     parser.add_argument("--test_dataset", nargs="+", default=[2695], type=int, help="list of dataset id")
     parser.add_argument("--tissue", default="Brain")  # TODO: Add option for different tissue name for train/test
     parser.add_argument("--train_dataset", nargs="+", default=[753], type=int, help="list of dataset id")
     parser.add_argument("--valid_dataset", nargs="+", default=[3285], type=int, help="list of dataset id")
-    parser.add_argument("--tune_mode", default="pipeline", choices=["pipeline", "params"])
+    parser.add_argument("--tune_mode", default="pipeline_params", choices=["pipeline", "params", "pipeline_params"])
     parser.add_argument("--seed", type=int, default=10)
-    parser.add_argument("--count", type=int, default=28)
-    parser.add_argument("--config_dir", default="", type=str)
+    parser.add_argument("--count", type=int, default=2)
     parser.add_argument("--sweep_id", type=str, default=None)
-    parser.add_argument("--result_name", default="best_test_acc.csv", type=str)
+    parser.add_argument("--summary_file_path", default="results/pipeline/best_test_acc.csv", type=str)
+    parser.add_argument("--root_path", default=str(Path(__file__).resolve().parent), type=str)
     args = parser.parse_args()
     logger.setLevel(args.log_level)
     logger.info(f"\n{pprint.pformat(vars(args))}")
-    MAINDIR = Path(__file__).resolve().parent
-    pipeline_planer = PipelinePlaner.from_config_file(f"{MAINDIR}/{args.config_dir}{args.tune_mode}_tuning_config.yaml")
+    file_root_path = Path(
```

Comment:
Currently, the name of the data set is used as the folder, and the name of the biological tissue can also be used.